transaction/builder/src/transaction_builder.rs

```diff
@@ -79,7 +80,7 @@ pub struct TransactionBuilder<FPR: FogPubkeyResolver> {
     /// The input materials used to form the transaction.
     input_materials: Vec<InputMaterials>,
     /// The outputs created by the transaction, and associated output secrets.
-    outputs_and_secrets: Vec<(TxOut, OutputSecret)>,
+    outputs_and_secrets: Vec<(TxOut, TxOutSummaryUnblindingData)>,
```

Comment:
This is the main change that is needed for the transaction builder for this to work